BlockSettleUILib/BSTerminalMainWindow.cpp

```diff
@@ -1077,7 +1079,30 @@ void BSTerminalMainWindow::initCcClient()
 void BSTerminalMainWindow::initUtxoReservationManager()
 {
    utxoReservationMgr_ = std::make_shared<bs::UTXOReservationManager>(
-      walletsMgr_, armory_, logMgr_->logger());
+            walletsMgr_, armory_, logMgr_->logger());
+}
+
+void BSTerminalMainWindow::initBootstrapDataManager()
+{
+   bootstrapDataManager_ = std::make_shared<BootstrapDataManager>(logMgr_->logger(), applicationSettings_);
```

Comment:
what's the difference between local file vs resource? are we talking in binary embedded .rc metadata?

Reply:
Yes. Local file - default source for latest version. If file missing - either cleanup or fresh install, so we need to grab embedded from resources and save to local file.